service/api_plugin_git.go

```diff
@@ -16,7 +16,11 @@ import (
 func (as *APIServer) gitServePatch(w http.ResponseWriter, r *http.Request) {
 	t := MustHaveTask(r)
```

Comment:
Do we need to do this if the patchParam is given?

Reply:
It'll always be there because the middleware checks the task for security purposes, but I moved things so that it's not always needed